From 1.2.0, gpgme will check if gpgconf & g13(from 1.3.0) exist, and cause
configure failure if not founding them in cross-compile environment. We can
add the gnupg-native to get them, but still have some issues:

1. need add new package: libksba, update libassuan(not in sato), and extend
   native support in some other packages(libgcrypt, libgpg-error, pth...)
2. Even we have gnupg, the g13 only exist in development branch, so that we
   still have the g13 check failure in configure.

I have searched the compile log, and found gpgconf/g13 are not used. So use a
simple method here, throw a warning mesg rather than an error here just like in
non-cross-compile environment.

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

Upstream-Status: Inappropriate [embedded specific]

Index: gpgme-1.3.0/configure.ac
===================================================================
--- gpgme-1.3.0.orig/configure.ac	2010-07-21 09:38:09.000000000 +0800
+++ gpgme-1.3.0/configure.ac	2010-07-21 09:39:19.000000000 +0800
@@ -574,7 +574,7 @@
 *** Could not find gpgconf, install gpgconf or use --with-gpgconf=PATH to enable it
 ***])
     else
-      AC_MSG_ERROR([
+      AC_MSG_WARN([
 ***
 *** Can not determine path to gpgconf when cross-compiling, use --with-gpgconf=PATH
 ***])
@@ -670,7 +670,7 @@
 *** Could not find g13, install g13 or use --with-g13=PATH to enable it
 ***])
     else
-      AC_MSG_ERROR([
+      AC_MSG_WARN([
 ***
 *** Can not determine path to g13 when cross-compiling, use --with-g13=PATH
 ***])
